src/components/PostList/index.module.scss

```diff
@@ -76,6 +76,21 @@
   font-family: colfax-web;
 }
 
+.photo,
+.noPhoto {
+  margin-right: 16px;
+  height: 40px;
+  width: 40px;
+  border-radius: 50%;
+}
+
+.noPhoto {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--c-white-1);
+}
```

Comment:
Could you add all these styles under `.avatar`? I'm wondering if you need two classes. A default/faalback `background-color` would be nice too.